xmtp_mls/src/groups/validated_commit.rs

```diff
@@ -628,6 +628,7 @@ fn extract_metadata_changes(
             immutable_metadata,
             old_mutable_metadata,
         ),
+        num_super_admins: old_mutable_metadata.super_admin_list.len() as u32,
```

Comment:
Shouldn't this be the `new_mutable_metadata`?

Reply:
Yea I had a check in a world where you can only remove 1 super admin at a time, that there must be more than 1 before you remove. But checking that the new mutable metadata has at least one super admin is the better option so that removing multiple superadmins would evaluate correctly as well. fixed here - d5d06f48829d0a7637ee67ffc16ea94deb220df1